Jenkinsfile

```diff
@@ -178,14 +178,19 @@ node {
     	    docker build $dockerCacheArg --build-arg pip_extra_index_params="--extra-index-url ${params.PyPiRepository}" --build-arg pip_legion_version_string="==${Globals.baseVersion}+${Globals.localVersion}" --build-arg source_image="legion/base-python-image" -t legion/k8s-edi .
     	    """
 
+    	    sh """
+    	    cd k8s/liara
```

Comment:
I think we need to rename is to console.

Reply:
Which name exactly? Do you mean just change all 'liara' to 'console'?